src/AVLTree.h

```diff
@@ -0,0 +1,135 @@
+//
+// Created by shvmo on 29/04/2020.
+//
+
+#ifndef DATASTRUCTURESWET1_AVLTREE_H
+#define DATASTRUCTURESWET1_AVLTREE_H
+
+#include "TNode.h"
+
```

Comment:
hmm plz look for any sequential new-lines around the project and remove 1 of them pl0x

Reply:
fiiixxxed